packages/grafana-ui/src/components/Button/Button.tsx

```diff
@@ -167,8 +171,22 @@ export const LinkButton = React.forwardRef<HTMLAnchorElement, ButtonLinkProps>(
       hasIcon: icon !== undefined,
     });
 
+    const linkButtonStyles =
+      disabled &&
+      cx(
+        disabledStyles,
+        css`
+          pointer-events: none;
+        `
+      );
+
     return (
-      <a className={cx(styles.button, className)} {...otherProps} ref={ref}>
+      <a
+        className={cx(styles.button, className, linkButtonStyles)}
```

Comment:
Correct me if I'm wrong, but if you put the className last it can overwrite previous styles? Maybe that's a desirable effect.
```suggestion
        className={cx(styles.button, linkButtonStyles, className)}
```

Reply:
I was thinking the same, but then decided we'd like the disabled styles to be enforced so they're not accidentally overridden, but now not sure about that 🤔